fiat-web/src/main/java/com/netflix/spinnaker/fiat/config/RetrofitConfig.java

```diff
@@ -33,37 +30,19 @@
 import org.springframework.context.annotation.Bean;
 import org.springframework.context.annotation.Configuration;
 import org.springframework.context.annotation.Primary;
+import org.springframework.http.converter.json.Jackson2ObjectMapperBuilder;
 import org.springframework.util.backoff.BackOffExecution;
 import org.springframework.util.backoff.ExponentialBackOff;
 import retrofit.RestAdapter;
 
-/** This package is placed in fiat-core in order to be shared by fiat-web and fiat-shared. */
+/** Provides base beans used for configuring Retrofit REST client facades. */
 @Configuration
 public class RetrofitConfig {
 
-  @Value("${ok-http-client.connection-pool.max-idle-connections:5}")
-  @Setter
-  private int maxIdleConnections;
-
-  @Value("${ok-http-client.connection-pool.keep-alive-duration-ms:300000}")
-  @Setter
-  private int keepAliveDurationMs;
-
-  @Value("${ok-http-client.retry-on-connection-failure:true}")
-  @Setter
-  private boolean retryOnConnectionFailure;
-
-  @Value("${ok-http-client.retries.max-elapsed-backoff-ms:5000}")
-  @Setter
-  private long maxElapsedBackoffMs;
-
   @Bean
   @Primary
-  ObjectMapper objectMapper() {
-    return new ObjectMapper()
-        .configure(DeserializationFeature.FAIL_ON_UNKNOWN_PROPERTIES, false)
```

Comment:
Intentionally re-enabling this?  AKA removing the disable would... then trigger failures on unknown properties?

Reply:
Per slack, the Jackson2ObjectMapperBuilder sets this by default, so by switching to it we're a touch cleaner.